README.md

```diff
@@ -35,6 +35,11 @@ sudo apt-get install git
 sudo apt-get install openjdk-7-jdk
 sudo apt-get install npm
 sudo apt-get install libfontconfig
+sudo npm -g install grunt-cli
```

Comment:
Technically, neither `node`, nor `npm` or
 grunt-cli https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-web/package.json#L29
 or bower  https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-web/package.json#L10
are required to be installed manually on the environment, as they all are beeing taken care of by `mvn package` in zeppelin-web through the [frontend-maven-pluging](https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-web/pom.xml#L88)